### DISCUSSION TITLE
R link screen information missing after failed update from dealer

### DISCUSSION THREAD

**Discussion by Nikoletapex** [cedricp/ddt4all#882] [Q&A]
Hi there..help please..anyone..i went my car to a renawlt dealer to upgrade system and maps..the dealer told me that the process got stuck two times and i left with problems at the screen..for example:it doesnt show information about the external temperature or the middle velocity and consumption as it used to display..the warning alarm when the car goes back ceased to work..the screen also doesnt show tyre pressure and others..its like he erased the potential to do any kind of measurements!apart from that the screen works, the speakers work ..i went the car back to the garage but he didnt kwow how to fix it..what do i do now?please please please help me im a woman and i dont know much of technology..

**Comment by matzel82** [comment]
They have to give a lot attention to a OTS.... there is a patch to install after update. Don't make self thinks to your R Link !!!!!

**Comment by matzel82** [comment]
I came from Germany and work in a Renault Garage too

**Comment by AndyIreland** [comment]
is it a rlink1 or 2? they done 2 versions.

This is rlink1 in the picture , does yours look like this?:
![r-link evolution](https://github.com/cedricp/ddt4all/assets/49172217/f91f6f4c-2ee1-47cd-8bd6-c9fe482d7986)

**Comment by AndyIreland** [comment]
> They have to give a lot attention to a OTS.... there is a patch to install after update. Don't make self thinks to your R Link !!!!!

what is an OTS ?

**Comment by Nikoletapex** [comment]
> I came from Germany and work in a Renault Garage too

Whats an OTS?Im irrelevant with tecnology so please use as simple words as possible..thank you for your response and as you mention you are a technician at renawlt,so tell me why did this happen to my screen and how can i fix it..i read a lot of cases of damaged screens and i i can tell by common logic that my screen is still functionable..its like the dealer has erased some information or the commands with his machine..and the answear he gave me to get rid of me was..you are lucky your screen didnt get black..he didnt know how to fix it it was obvious..any ideas?

**Comment by Nikoletapex** [comment]
> is it a rlink1 or 2? they done 2 versions.
> 
> This is rlink1 in the picture , does yours look like this?: ![r-link evolution](https://user-images.githubusercontent.com/49172217/256979581-f91f6f4c-2ee1-47cd-8bd6-c9fe482d7986.jpg)

Ive sent a picture of the screen.did you see it?thank you for your reply

**Comment by AndyIreland** [comment]
have you got a ELM327 OBD2 diagnostic adaptor @Nikoletapex  ?

**Reply by Nikoletapex** [reply]
I dont have any kind of equipment or knowledge as you do thats why i went to the garage!the thing for sure is that the update didnt happen at all, as the technician told me ,and the procedure got stuck two times and he quit trying..i'll take the car on monday to thenext nearest city that has an authorised renawlt garage but im sure they wont know neither!the only clues i can show them are your information and thank you very much for your time..but its weird...the update didnt happen..how come some measurements stop working?maybe its a virus or he tried to put a wrong version of software that is not supported?so what do you think i could say to the dealer to help him fix it?
What should he do?im sure you know more than him its obvious from the way you refer to details..

**Reply by AndyIreland** [reply]
not really, i dont know a lot about the rlink2 (apart from a lot of people have 'bricked' =broken = black screen) through updating badly. 

All you could do whoever you leave it into is point them to these 2 websites that might be of help to them:

https://www.gps-rlink.com/r-link-2/update-software/

https://www.drive2.ru/l/610934291198664970/

if they are a proper renault main dealer though I would have thought they would know how to update it properly though without 'bricking it' 

Yeah, its a shame you have not got a ELM327 diagnostics adaptor and a little bit of technical knowledge because although I or you would not be able to update it to later versions it would have meant that maybe we could have enabled the features that you reckon are not working since you left it into the last garage  - but no chance of that if you havent got the adaptor

**Reply by Nikoletapex** [reply]
Thank you very much for the links you sent me..i left the car at the garage because the technician was on vacation and im waiting for their call..i'll let you know what they will tell me about 'the diagnose'!thanks again!

**Reply by AndyIreland** [reply]
good luck - hope you get it sorted

**Comment by AndyIreland** [comment]
@Nikoletapex if that was 'REALLY' updated , your garage would have updated it to version 3.3 so you could get Android Auto on it (but then going from 2.2 to 3.3 can cause black screen if not done properly or not taken to a garage to have the update done as far as I understand it?)

if you had an ELM327 OBD2 diagnostic adaptor and specially wired OBD2 cable to read the rlink through the OBD2 socket (you need to have specially wired cable for rlink1, I am not sure whether you need it for rlink2 too?) you (or I could remotely) could just see if outside temp and other parameters are enabled after they did what they did and if not may be able to be enabled again using DDT4ALL or DDT4ALL mode through PYREN

If they were using PYREN software (which I wouldnt have thought so, I would have thought they used CAN CLIP software) but if they did use PYREN , if they done it properly they would have (Should have) backed up/made a dump file of your rlink2 and its settings so that if anything went wrong they could load the dump file back on and get it to how it was.

**Comment by BarektE** [comment]
I solved the same problem just yesterday. If robroc is performed by the authorized service, 1 map will be destroyed.
2 outside temperature disappears
3 backup cameras disappear
4 parking assist assistants disappear
After the service update, find another software from the internet and install it and it will be fixed. In fact, when installing the service, errors occur if you do not plug in the USB, which is the infrastructure of these devices.
